src/connectors/numeric-selector/__tests__/connectNumericSelector-test.js

```diff
@@ -235,4 +235,57 @@ describe('connectNumericSelector', () => {
       refine = renderingParameters.refine;
     });
   });
+
+  it('The refine function can unselect with `undefined` and "undefined"', () => {
+    const rendering = sinon.stub();
```

Comment:
Prefer usage of `jest.fn`, at some point we should remove `sinon`

Reply:
Yes but if I move to jest.fn, I need to do the whole file. You might say "yes exactly do it and move like that each time you encounter sinon" and I would have to agree. OK :)